hack-agenda.md

```diff
@@ -0,0 +1,14 @@
+#Hack-a-thon Agenda
+
+##Thursday
+- What is victory, clarify the vision and the first milestone targets
+- API design generally, detailed API design for foundational components
+- Animation Model!
+- Take stock of what we already have / can reuse
+- Hack
+
+##Friday
+- Talk to Paula about base styling in Radium
+- Revisit API design post hacking
+- Brainstorm cool capstone project based on github data
```

Comment:
Chord is a contender for this - will demo that and talk about conversion, that will be fertile source of conversation.